assets/php/class-wgpb-block-grid-base.php

```diff
@@ -216,7 +215,6 @@ public function render() {
 	protected function get_container_classes() {
 		$classes = array(
 			'wc-block-grid',
-			"wp-block-{$this->block_name}",
```

Comment:
@ryelle Not sure if you wanted to keep this or not. I understand it's a WP core pattern but having both classes (wc and wp) seemed a bit redundant. Thought `wc` prefix may be preferred. let me know :)

Reply:
The `wp-block-` class is a core default, so anyone styling blocks will probably expect to see this class name.